Spanner/src/Connection/Grpc.php

```diff
@@ -455,6 +457,39 @@ public function createSession(array $args)
         ]);
     }
 
+    /**
+     * @return PromiseInterface
+     */
+    public function createSessionAsync(array $args)
+    {
+
+        $database = $this->pluck('database', $args);
+        $opts = $this->addResourcePrefixHeader([], $database);
+        $opts['credentialsWrapper'] = $this->credentialsWrapper;
+        $transport = $this->spannerClient->getTransport();
+
+        $request = new CreateSessionRequest([
+            'database' => $database
+        ]);
+
+        $session = $this->pluck('session', $args, false);
+        if ($session) {
+            $sessionMessage = $this->serializer->decodeMessage(new Session, $session);
+            $request->setSession($sessionMessage);
+        }
+
+        $promise = $transport->startUnaryCall(
```

Comment:
WDYT of dropping the return on line 490 and just returning here?

```suggestion
        return $transport->startUnaryCall(
```

Reply:
I can dig it!